terraform/modules/ops/main.tf

```diff
@@ -154,3 +154,68 @@ resource "google_cloud_scheduler_job" "nightly_schedule" {
     time_sleep.wait_for_artifactregistry
   ]
 }
+
+############################
```

Comment:
Same file-level comment as above: let's isolate this to a delivery_testing.tf for now while we sort out where these resources should properly live.

Reply:
Done.